src/kubetop/_textrenderer.py

```diff
@@ -369,8 +369,8 @@ def _render_container(container):
 
 def partition(seq, pred):
     return (
-        filter(pred, seq),
-        filter(lambda x: not pred(x), seq),
+        u"".join([x for x in seq if pred(x)]),
+        u"".join([x for x in seq if not pred(x)]),
```

Comment:
Can you use `list(...)` instead of `[...]` to avoid leaking the loop variable on Python 2?